src/client/components/admin/UserContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import Users from './Users';
+import UserTableRow from './UserTableRow';
+
+class UserContainer extends React.Component< {}, {
+    userData: Array<React.ReactElement<HTMLDivElement>>} > {
+    constructor(props: any) {
+        super(props);
+
+        this.state = {
+            userData: [],
+        };
+    }
+
+    public async componentDidMount(): Promise<void> {
```

Comment:
wrong lifecycle hook, see review comment from before - this should be componentWillMount because you're rendering the page, and then immediately rerendering it..